docs/python/tutorials/classes.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 15
+title: Decorating Class Methods
```

Comment:
"Class methods" can cause confusions here. Because it is a special term in Python. Can we change it to "Adding DBOS to Python Classes" or "Working with Python Classes"?